src/Datadog.Trace.ClrProfiler.Native/environment_variables_util.h

```diff
@@ -47,7 +47,14 @@ bool EnableInlining(bool defaultValue) {
 }
 
 bool IsCallTargetEnabled() {
+#if defined(ARM64) || defined(ARM)
+  //
+  // If the architecture is ARM64 or ARM, we enable CallTarget instrumentation by default
+  //
+  ToBooleanWithDefault(GetEnvironmentValue(environment::calltarget_enabled), true);
+#else
   CheckIfTrue(GetEnvironmentValue(environment::calltarget_enabled));
+#endif
```

Comment:
Can we change this for consistency? It will be easier to follow the logic if they both use `ToBooleanWithDefault()`
```suggestion
  ToBooleanWithDefault(GetEnvironmentValue(environment::calltarget_enabled), true);
#else
  ToBooleanWithDefault(GetEnvironmentValue(environment::calltarget_enabled), false);
#endif
```

Reply:
Sure, will do.

Reply:
done in 3684467